spec/models/pull_request_spec.rb

```diff
@@ -30,12 +30,12 @@
   end
 
   describe ".needs_reposting" do
-    it "returns PRs that haven't been reviewed in over 30 minutes nor reposted" do
+    it "returns PRs that haven't been reviewed in over the time threshold nor reposted" do
       pr_to_report = create(:pull_request, :needs_review, reposted: false)
       _reposted_pr = create(:pull_request, :needs_review, reposted: true)
       _pr_in_progress = create(:pull_request, :in_progress)
 
-      travel(31.minutes) do
+      travel(repost_threshold.minutes + 1) do
```

Comment:
What do you think about using our gem to manage the ENV here and make sure it is set for the test?